keras_cv/models/stable_diffusion/text_encoder.py

```diff
@@ -37,6 +37,26 @@ def __init__(self, max_length, vocab_size=49408, name=None, download_weights=Tru
             self.load_weights(text_encoder_weights_fpath)
 
 
+class TextEncoderV2(keras.Model):
+    def __init__(self, max_length, vocab_size=49408, name=None, download_weights=True):
+        tokens = keras.layers.Input(shape=(max_length,), dtype="int32", name="tokens")
+        positions = keras.layers.Input(
+            shape=(max_length,), dtype="int32", name="positions"
+        )
+        x = CLIPEmbedding(vocab_size, 1024, max_length)([tokens, positions])
+        for _ in range(23):
+            x = CLIPEncoderLayer(1024, 16, use_quick_gelu=False)(x)
+        embedded = keras.layers.LayerNormalization(epsilon=1e-5)(x)
+        super().__init__([tokens, positions], embedded, name=name)
+
+        if download_weights:
+            text_encoder_weights_fpath = keras.utils.get_file(
+                origin="https://huggingface.co/Jobayer/stable_diffusion_v2/resolve/main/text_encoder_v2_1.h5",
+                file_hash="985002e68704e1c5c3549de332218e99c5b9b745db7171d5f31fcd9a6089f25b"
```

Comment:
need an export weight from our public GCP bucket

Reply:
I think instead of a GCS bucket we should just use HF weights to keep this separate from our corp infra (like what Francois' did with the original weights).

I will make a clone of these weights in a HF account that we own.

Reply:
Updated.